jenkins/ci.suse.de/pipelines/openstack-cloud-heat.Jenkinsfile

```diff
@@ -67,8 +70,8 @@ pipeline {
           // Dump the heat_template multi-string parameter value into a file
           writeFile file: "$WORKSPACE/heat_template.yml", text: params.heat_template
 
-          lock(resource: 'cloud-ECP-API') {
-            timeout(time: 10, unit: 'MINUTES', activity: true) {
+          lock(resource: "$os_cloud-API") {
+            timeout(time: 20, unit: 'MINUTES', activity: true) {
```

Comment:
@flaviodsr because we use the same Jenkinsfile for both ECP and SUSE cloud platforms, and the SUSE cloud platform is slower (based on Newton, whereas ECP is Pike).